bpf/.cargo/config.toml

```diff
@@ -0,0 +1,15 @@
+# We have this so that one doesn't need to manually pass
+# --target=bpfel-unknown-none -Z build-std=core when running cargo
+# check/build/doc etc.
+#
+# NB: this file gets loaded only if you run cargo from the bpf/
+# directory, it's ignored if you run from the workspace root. See
+# https://doc.rust-lang.org/cargo/reference/config.html#hierarchical-structure
+#
+# NB-2: cargo xtask docs currently relies on this file existing.
+[build]
+target-dir = "../target"
```

Comment:
i think this can be omitted; it's the default behavior regardless of where you are in tthe workspace